enterprise/server/api/api_server_test.go

```diff
@@ -21,80 +22,113 @@ import (
 var userMap = testauth.TestUsers("user1", "group1")
 
 func TestGetInvocation(t *testing.T) {
+	testUUID, err := uuid.NewRandom()
+	assert.NoError(t, err)
```

Comment:
for errors during setup you should use `require.NoError` which immediately terminates the test unlike `assert.NoError` which allows the test to continue even if the test setup failed

Reply:
Done